test/cases/export-default/input.js

```diff
@@ -1,2 +1,2 @@
-const foo = "123";
+var foo = "123";
```

Comment:
Don't modify `input.js`. This change doesn't require the input to be changed. Same as all other `input.js` files.